modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -23,6 +23,14 @@ and the `bmc` parameter for the `install-config.yaml` file.
 | `UEFI`
 | The boot mode for a node. Options are `legacy`, `UEFI`, and `UEFISecureBoot`. If `bootMode` is not set, Ironic sets it while inspecting the node.
 
+| `bootstrapExternalStaticIP`
+|
+| The static IP address for the bootstrap VM. You must set this value when deploying a cluster with static IP addresses.
```

Comment:
> You must set this value when deploying a cluster with static IP addresses.

I think the formulation here is a little unclear. I think it should be stated that this field must be set when there is no DHCP service in the environment.

Reply:
Thank you for your comment. I've updated the document accordingly.